extensions/wikia/AdEngine/js/video/videoSettings.js

```diff
@@ -36,6 +38,8 @@ define('ext.wikia.adEngine.video.videoSettings', [
 				return true;
 			}
 
+			return true;
+
 			if (sampling > 0) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this code after the "return" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnreachableCode)